scripts/select-random-player/select_random_player.lua

```diff
@@ -0,0 +1,30 @@
+local BE_SPECIAL = "Traitor"
+
+function selectRandomPlayer(be_special)
```

Comment:
Casing should be camelCase (unless we haven't been doing that for whatever reason, I have)

Reply:
Yeah this is what I've been doing